### PR TITLE
python-chado: fix wrong param name in load_gff

### PR DESCRIPTION
**Pull request by abretaud**
A fix for a little bug I found while using the latest tool update...

### CODE REVIEW

**Comment by abretaud** [comment]
Oh, hang on, I think I found another (unrelated) bug in python-chado itself, I might need to make a new release...

**Comment by abretaud** [comment]
@erasche it should be ready now if you have the time to reapprove

File: tools/chado/feature_load_fasta.xml [hunk 1]
```diff
@@ -60,8 +60,12 @@ $do_update
 			label="Analysis" />
 		<param name="sequence_type" label="Sequence Type" type="text" value="contig" help="this should be a Sequence Ontology term"/>
 
-		<param name="re_name" label="Name regular expression" argument="re_name" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true" />
-		<param name="re_uniquename" label="Uniquename regular expression" argument="re_uniquename" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true" />
+		<param name="re_name" label="Name regular expression" argument="re_name" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true">
+            <expand macro="sanitized"/>
```
Comment (bgruening) [inline tools/chado/feature_load_fasta.xml]: indentation ...

File: tools/chado/feature_load_fasta.xml [hunk 2]
```diff
@@ -60,8 +60,12 @@ $do_update
...
+        </param>
+		<param name="re_uniquename" label="Uniquename regular expression" argument="re_uniquename" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true">
+            <expand macro="sanitized"/>
```
Comment (bgruening) [inline tools/chado/feature_load_fasta.xml]: indentation ...

File: tools/chado/feature_load_fasta.xml [hunk 3]
```diff
@@ -60,8 +60,12 @@ $do_update
 			label="Analysis" />
 		<param name="sequence_type" label="Sequence Type" type="text" value="contig" help="this should be a Sequence Ontology term"/>
 
-		<param name="re_name" label="Name regular expression" argument="re_name" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true" />
-		<param name="re_uniquename" label="Uniquename regular expression" argument="re_uniquename" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true" />
+		<param name="re_name" label="Name regular expression" argument="re_name" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true">
+            <expand macro="sanitized"/>
+        </param>
+		<param name="re_uniquename" label="Uniquename regular expression" argument="re_uniquename" type="text" help="Regular expression to extract the feature name from the fasta sequence id (first capturing group will be used)." optional="true">
```
Comment (bgruening) [inline tools/chado/feature_load_fasta.xml]: the argument should be `--re_uniquename` also the others.

File: tools/chado/feature_load_gff.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="feature_load_gff" name="Chado load gff" version="@WRAPPER_VERSION@.0">
+<tool id="feature_load_gff" name="Chado load gff" version="@WRAPPER_VERSION@.1">
```
Comment (bgruening) [inline tools/chado/feature_load_gff.xml]: this is not needed, as we bump the entire version in the macros isn't it?

**Comment by abretaud** [comment]
Thanks, it should be better now, I've reviewed the indentation + argument="--xx" everywhere

**Comment by abretaud** [comment]
Any time for another review/approval? (no hurry)

**Comment by erasche** [comment]
@bgruening is it ok for you?

**Comment by abretaud** [comment]
Hey! Is it ok to merge?

**Comment by bgruening** [comment]
On it.

**Comment by bgruening** [comment]
LGTM!

**Pull request merged**